src/lib/localization.js

```diff
@@ -0,0 +1,52 @@
+import { L10nError } from './errors';
+
+export class Localization {
```

Comment:
Possibly.  I'm still considering renaming this to `FallbackStrategy` which is used by `HTMLLocalization`.  However, I think regular functions will work jus fine in the current scenario, since we only have one strategy anyways for now.  I'll change it.